.github/workflows/release-drafter.yml

```
@@ -10,6 +10,10 @@ on:
 permissions:
   contents: read
 
+concurrency: 
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
```

```suggestion
jobs:
```
Feel free to give thoughts on this, but... I kind of don't want to cancel the release drafter?

---

It doesn't really as it only takes about 15s, but it's safe to cancel as we only need it to run once: 

when it runs it looks at the current state of merged PRs and overwrites the previous draft.

---

Although two different runs from two different merges will have different `${{ github.ref }}` so won't be cancellable.

However, manual triggers via `workflow_dispatch` could have the same `ref`. Again, no strong opinion either way for this.